### PR TITLE
Changed third sentence in first paragraph of 3.1 Upgrading Satellite …

### PR DESCRIPTION
**Pull request by bangelic**
…Server per ticket SATDOC-610
@melcorr 

Cherry-pick into:

* [ ] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 This section describes how to upgrade {ProjectServer} from {ProjectVersionPrevious} to {ProjectVersion}.
 You can upgrade from any minor version of {ProjectName} Server {ProjectVersionPrevious}.
 ifdef::satellite[]
-However, to access the latest migration and tooling fixes, upgrade to the latest minor version of {ProjectServer} before upgrading from {ProjectVersionPrevious} to {ProjectVersion}.
+However, to access the latest migration and tooling fixes, upgrade to the latest 6.9.z before starting migration to {ProjectVersion}.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc:7]: We don't ever put version numbers into the docs. That's why we were using {ProjectVersionPrevious} here.

Comment (bangelic) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc:7]: I might need to get more information on what version they specifically want to use. In the Jira ticket, it says **"The text says that you can upgrade from any version of 6.9 to 6.10. However, fixes for the migration were added to satellite in 6.9.7 and possibly 6.9.8. The documentation should be changed to say that the customer needs to upgrade to the latest 6.9.z before upgrading to 6.10.**

Comment (melcorr) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc:7]: "Ensure that you have updated to the latest z-stream before upgrading to 6.10. For more information about updating {ProjectServer}, see xref: updating reference here."

That will cover you.

Comment (bangelic) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc:7]: Thank you!

**Review by melcorr — APPROVED** [review]
LGTM! Thanks.

**Comment by melcorr** [comment]
@bangelic please learn to write commit messages!

**Pull request merged**